bluesign/vscode-cadence
# Support for new emulator debugger

- Fix test cases
- Create initialization test and fix account data
- Delete flow cli
- Add updated flow cli
- Run e2e-test
- Update README.md
- Add build workflow
- Update README.md
- Run package
- Update ci
- Update tests
- Add integration test to ci
- Fix spacing
- Update ci
- Check flow-cli
- Docker enable flow permissions
- Add Sentry to dependencies
- Docker check flow
- Update README.md
- Update README.md
- Initialize docker flow-cli
- Test sudo
- fix test sudo
- Try -c command
- Try CI on windows
- Keep flow
- Run macos
- update docker compose call
- Ensure flow-cli can be executed
- tests-passing-locally
- Add Sentry Dependency
- Update package
- Update windows paths
- Updae package-lock
- Remove e2e tests from CI
- Add Windows compile test
- Integrade hosted emulator
- Remove AccountData unit test
- Add deactivate
- Remove commands deactivate
- Update response
- Start implementing installers
- Start implementing installers
- LS initialization options
- Flow CLI Installer
- Add flow-cli installs for mac/ windows
- Update installer
- Update
- Try to install homebrew
- Add dependency installers
- Create Dependency Installer
- Fix lint
- Fix initialization options
- Move prompts to UI
- Update Dependency Installers
- Update prompts
- Copy active account to clipboard
- Copy active account to clipboard
- Copy active account to clipboard
- Update settings
- Remove start/ stop emulator commands
- Fix initialization
- Updated
- Copt active account to clipboard
- pass number of accounts as string
- pass number of accounts as string
- Remove start/ stop emulator commands from e2e tests
- Test responses
- test CI
- test flow
- test flow
- test flow
- test ls
- test cypress
- test
- test
- test docker
- try docker
- Add flow binary for macos
- test
- re-test
- test no video
- test ubuntu
- Test new flow cli
- change flow-cli
- run macos-12
- Convert json obj to accounts
- Add back telemetry
- Update responses
- Remove restart
- Update send request
- Update CHANGELOG
- Add restart command
- add users
- test cypress
- add jobs
- test
- test
- test
- Update version for release
- Move codelens utils
- test
- test
- test e2e-test
- record cypress
- add record key
- test
- Package test
- Wait for container
- test macos
- test cypress
- Run cypress
- test
- test video
- Test macos
- setup docker macos
- test cypress
- Install flow-cli
- test
- remove password
- vscode exec
- test export path
- test sh
- test
- full test
- add steps
- full ci
- End To End Testing Now Working
- Update README.md
- Remove docker path export
- rename
- rename ci
- wait for it
- test wait for it action
- Wait for docker container
- Update README.md
- Update e2e tests for hosted emulator
- Add integration tests to CI
- test windows
- Update lint
- test import
- test
- test checkout
- test
- test
- Setup ci
- update ci layout
- Update Cypress to 10.0.0
- Update cypress config
- Always trust workspace folder
- test cypress before
- Keep cypress 8.3.0
- install flow-cli in package cmd
- Import Mocha for integration tests
- update
- Test integration on CI
- Install mocha
- Test integration on macos
- test macos
- Update unit tests to mocha
- Update package
- Generate UID and report version number
- Add version statistics with unique uid
- Generate uuid
- Remove test launch
- Fix integration tests
- Remove unused dependency
- Update e2e tests
- Update package-lock
- Add uuid dependency
- Update uuid
- Update package-lock
- update uuid
- Update package
- Update integration test paths
- test
- Test uuid
- update package lock
- test ci
- Update integration paths
- test ubuntu
- Disable telemetry on integration tests
- Update integration test
- Update package-lock
- Update CHANGELOG for v1.0.0
- Track activations using mixpanel
- Track activations using Mixpanel
- Update CHANGELOG
- Set user operating system
- Add user operating system to mixpanel
- Add user information to mixpanel
- Add user information to mixpanel
- Update CHANGELOG
- Update Mixpanel Token
- Fix codelens refresh
- Fix codelens refresh
- Upsate user info
- Update mixpanel info
- Update mixpanel information
- Update user information
- Add activation event
- Create event enums
- Add information messages
- Add flow-cli version check
- Add error tracking to mixpanel
- Add error reporting to mixpanel
- Update e2e-test flow-cli to v0.39.2
- Fix e2e tests for v1.0.0 release
- Update init extension helper
- Update package.json
- Ignore syntax errors in Sentry
- Send telemetry errors to Mixpanel
- Catch exceptions with Mixpanel
- Send errors to mixpanel
- Version 1.0.3
- Capture unhandled exceptions of any type
- Disable Sentry auto tracking
- v1.0.4
- Add ovsx dependency
- Create workflow to publish extension
- Clean up dev dependencies
- Update bug-report.md
- Update feature-request.md
- Flow-cli dependency v0.40.0
- fix path to language configuration
- Create codeql.yml
- Parse version string
- Fix switch account e2e test
- Update switch accounts test
- Use ts-mocha for unit tests
- fix lint
- Bump minimatch from 3.0.4 to 3.1.2
- Update publish workflow
- Bump qs from 6.5.2 to 6.5.3
- Add Windows build to ci
- Add windows npm commands
- Update ci
- Add windows commands
- Bump json5 from 2.2.0 to 2.2.3
- Add installer popups
- Update required Flow CLI version
- Fix lint
- Bump release to v1.0.6
- Upate popup message
- Update platform to node os
- Update node
- Fix lint
- Update ps install script
- Update bash install cmd
- Check latest flow-cli version
- Update string
- fix lint
- Find latest flow-cli version
- update language client
- Update version parser
- Add LS started popup
- Enable Windows flow-cli update
- Fix lint
- Configure dependabot
- Update readme
- Update README.md
- Update README.md
- Update README.md
- Update README
- Rename gif
- Update README.md
- Code gen gif
- debugger with new emulator debug server

Closes #???

## Description



______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Added appropriate labels
